### ISSUE TITLE
no_unnecessary_double_quotes false negative

### ISSUE THREAD

**Issue by drewinglis** [clutchski/coffeelint#300]
The following doesn't trigger the "no_unnecessary_double_quotes" rule, but it should:

```
strangeFn = (message) ->
  message.text =
    """
    Here is some text which doesn't seem to trigger coffeelint.
    """
```

I'm using coffeelint version 1.5.2. Let me know if there's any other information I can provide. Thanks!
